apps/hyperdrive-trading/src/ui/markets/PoolDetails/PoolDetails.tsx

```diff
@@ -48,16 +51,49 @@ export function PoolDetails({
           <CustomBanner description="This market has been paused. You may close your positions, but no new positions may be opened." />
         )}
       </div>
-      {(() => {
-        switch (activePosition) {
-          case "longs":
-            return <OpenLongForm2 hyperdrive={hyperdrive} />;
-          case "shorts":
-            return <OpenShortForm2 hyperdrive={hyperdrive} />;
-          case "lp":
-            return <AddLiquidityForm2 hyperdrive={hyperdrive} />;
-        }
-      })()}
+      <div className="flex flex-col gap-16">
+        {(() => {
+          switch (activePosition) {
+            case "longs":
+              return <OpenLongForm2 hyperdrive={hyperdrive} />;
+            case "shorts":
+              return <OpenShortForm2 hyperdrive={hyperdrive} />;
+            case "lp":
+              return <AddLiquidityForm2 hyperdrive={hyperdrive} />;
+          }
+        })()}
+
+        <FAQ />
+      </div>
+    </div>
+  );
+}
+function FAQ() {
+  return (
+    // Add mx-2 so the FAQ lines up with the rest of the form
+    <div className="mx-2 flex flex-col gap-6">
+      <h4>FAQ</h4>
+
+      <div className="flex flex-col gap-6">
+        {faqData2.map(({ question, answer }, i) => (
+          <AccordionSection2
+            key={i}
+            heading={
+              <div
+                className={classNames(
+                  "pb-1 text-left text-md font-normal text-base-content/80 hover:text-white group-data-[open]:text-white",
+                )}
+              >
+                {question}
+              </div>
+            }
+          >
+            <div className="flex flex-col gap-4 text-md leading-bodyText text-base-content/80">
+              {answer}
```

Comment:
My editor is saying text-md applies the same classes as text-bodyText
```suggestion
            <div className="flex flex-col gap-4 text-md text-base-content/80">
```